internal/service/service.go

```diff
@@ -0,0 +1,17 @@
+package service
+
+import (
+	"github.com/arturzhamaliyev/Flight-Bookings-API/internal/repository"
+)
+
+// Service provides user-related functionality
+type Service struct {
+	User *Users
+}
+
+// New creates a new Service with the given repository.
+func New(usersRepo *repository.UsersRepository) Service {
```

Comment:
what I want to change my db from posrgre to mongo?
how I can replace them with minimal changes in this code?